retrofit/lib/annotations/annotations.dart

```diff
@@ -2,5 +2,6 @@ export 'requests.dart';
 export 'params.dart';
 
 class GenApiClient {
-  const GenApiClient();
+  final String basePath;
+  const GenApiClient([this.basePath = ""]);
```

Comment:
Please make `basePath` named optional argument.

Reply:
Do you mean just initialize it to null instead ?